db/seeds.rb

```diff
@@ -1,7 +1,52 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+
+puts "Clearing out the battlefield 🌋..."
+User.destroy_all
+PlayerGroup.destroy_all
+Group.destroy_all
+Preference.destroy_all
+Rulebook.destroy_all
+Genre.destroy_all
+puts "Battlefield Cleared ⛰"
+
+puts "Summoning Gandalf 🧙🏻‍♂️"
+gandalf = User.new
```

Comment:
better to use `gandalf = User.create!()` and the put the lines below inside the parentheses